docs/runbooks/KubeMacPoolDown.md

```diff
@@ -47,3 +47,5 @@ If you cannot resolve the issue, see the following resources:
 - [#virtualization Slack channel](https://kubernetes.slack.com/channels/virtualization)
 <!--USend-->
 
+**Note:** Starting from 4.14, this runbook was replaced by [KubemacpoolDown runbook](http://kubevirt.io/monitoring/runbooks/KubemacpoolDown.html).
```

Comment:
Sorry, I think we should move this below the alerts name at the top of the runbook. WDYT?

Reply:
sounds good. should I move the note in https://github.com/kubevirt/monitoring/blob/main/docs/runbooks/SSPOperatorDown.md#mitigation as well?

Reply:
I don't think that will be necessary. We would normally mention this in an RN.

Reply:
Done only for this alert.